django/views/csrf.py

```diff
@@ -131,4 +130,8 @@ def csrf_failure(request, reason=""):
         'docs_version': get_docs_version(),
         'more': _("More information is available with DEBUG=True."),
     })
+    try:
+        t = loader.get_template(template_name)
+    except TemplateDoesNotExist:
+        t = Engine().from_string(CSRF_FAILURE_TEMPLATE)
```

Comment:
Yes, thanks for pointing that out. I'll create a ticket to make sure those error views are also changed. In the meanwhile I suggest we do it the right way here.